vdr/didsubject/manager_test.go

```diff
@@ -420,3 +420,20 @@ func (t testMethod) Commit(_ context.Context, _ orm.DIDChangeLog) error {
 func (t testMethod) IsCommitted(_ context.Context, _ orm.DIDChangeLog) (bool, error) {
 	return t.committed, t.error
 }
+
+func Test_sortDIDDocuments(t *testing.T) {
+	t.Run("duplicate", func(t *testing.T) {
+		documents := []did.Document{
+			{ID: did.MustParseDID("did:example:1")},
+			{ID: did.MustParseDID("did:example:1")},
+			{ID: did.MustParseDID("did:test:1")},
+		}
+
+		sortDIDDocuments(documents, []string{"test", "example"})
+
+		require.Len(t, documents, 3)
+		assert.Equal(t, "did:test:1", documents[0].ID.String())
+		assert.Equal(t, "did:example:1", documents[1].ID.String())
+		assert.Equal(t, "did:example:1", documents[2].ID.String())
```

Comment:
bad test
the sorted results contain compared to the original list documents [2,0,0], document on original index [1] has disappeared. (possible scenario where this may fail in the future (so godoc is fine, but this test is not) is when the list contains different versions of a DID document

Reply:
now failing after I made some changes

Reply:
This is not a valid use case for the function, I made its description more precise to avoid future mistakes about its usage.